src/schemapack/_internals/dump.py

```diff
@@ -17,14 +17,15 @@
 """Logic for dumping schemapacks."""
 
 import json
+from io import StringIO
 from pathlib import Path
 from typing import Any
 
 import ruamel.yaml
 
 from schemapack._internals.spec.schemapack import SchemaPack
 
-yaml = ruamel.yaml.YAML(typ="safe")
+yaml = ruamel.yaml.YAML(typ="rt")
```

Comment:
Do we still need to set `default_flow_style=False` then?

Reply:
Tests also run without. I removed it.